xtext/Dockerfile

```diff
@@ -3,6 +3,8 @@
 
 FROM node:19-bullseye AS toolstaticbuild
 
+ARG TRUSTED_ORIGINS
```

Comment:
This would benefit from an explanation and documentation update. What needs to be the value of `TRUSTED_ORIGINS`?

Reply:
I recommend adding a line of comment in the Dockerfile and explain in more detail in. the [educationplatform-docker](https://github.com/mdenet/educationplatform-docker). What do you think?

Reply:
Yes, would be good to have this in both places as not everyone might run the platform via the `-docker` repository.

Reply:
```suggestion
# comma delimited whitelist of base URLs (with no trailing slash) for CORS aware endpoints
ARG TRUSTED_ORIGINS
```